cdap-proto/src/main/java/co/cask/cdap/proto/RestartServiceInstancesStatus.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.proto;
+
+import java.util.Set;
+
+/**
+ * Class to describe status of restart instances of service request.
+ */
+public class RestartServiceInstancesStatus {
+  private final Set<Integer> instanceIds;
```

Comment:
Makes sense to me. Doesn't make sense to have multiple instances of the same ID in a restart status.